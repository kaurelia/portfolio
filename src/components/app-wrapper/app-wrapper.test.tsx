import { render } from "@testing-library/react";
import AppProvider from "../app-provider/app-provider";
import AppWrapper from "./app-wrapper";

describe("Test app-wrapper component", () => {
  it("Check if app-wrapper renders properly", async () => {
    const { container } = render(
      <AppProvider>
        <AppWrapper />
      </AppProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
