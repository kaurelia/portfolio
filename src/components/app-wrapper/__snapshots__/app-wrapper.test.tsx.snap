// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test app-wrapper component Check if app-wrapper renders properly 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
  height: 100vh;
  position: relative;
  overflow: hidden;
}

.emotion-1 {
  position: absolute;
  font-size: 1.5rem;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  pointer-events: none;
  top: inherit;
  left: inherit;
}

.emotion-2 {
  -webkit-transform: rotate(
      0deg
    );
  -moz-transform: rotate(
      0deg
    );
  -ms-transform: rotate(
      0deg
    );
  transform: rotate(
      0deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-3 {
  -webkit-transform: rotate(
      22.5deg
    );
  -moz-transform: rotate(
      22.5deg
    );
  -ms-transform: rotate(
      22.5deg
    );
  transform: rotate(
      22.5deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-4 {
  -webkit-transform: rotate(
      45deg
    );
  -moz-transform: rotate(
      45deg
    );
  -ms-transform: rotate(
      45deg
    );
  transform: rotate(
      45deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-5 {
  -webkit-transform: rotate(
      67.5deg
    );
  -moz-transform: rotate(
      67.5deg
    );
  -ms-transform: rotate(
      67.5deg
    );
  transform: rotate(
      67.5deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-6 {
  -webkit-transform: rotate(
      90deg
    );
  -moz-transform: rotate(
      90deg
    );
  -ms-transform: rotate(
      90deg
    );
  transform: rotate(
      90deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-7 {
  -webkit-transform: rotate(
      112.5deg
    );
  -moz-transform: rotate(
      112.5deg
    );
  -ms-transform: rotate(
      112.5deg
    );
  transform: rotate(
      112.5deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-8 {
  -webkit-transform: rotate(
      135deg
    );
  -moz-transform: rotate(
      135deg
    );
  -ms-transform: rotate(
      135deg
    );
  transform: rotate(
      135deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-9 {
  -webkit-transform: rotate(
      157.5deg
    );
  -moz-transform: rotate(
      157.5deg
    );
  -ms-transform: rotate(
      157.5deg
    );
  transform: rotate(
      157.5deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-10 {
  -webkit-transform: rotate(
      180deg
    );
  -moz-transform: rotate(
      180deg
    );
  -ms-transform: rotate(
      180deg
    );
  transform: rotate(
      180deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-11 {
  -webkit-transform: rotate(
      202.5deg
    );
  -moz-transform: rotate(
      202.5deg
    );
  -ms-transform: rotate(
      202.5deg
    );
  transform: rotate(
      202.5deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-12 {
  -webkit-transform: rotate(
      225deg
    );
  -moz-transform: rotate(
      225deg
    );
  -ms-transform: rotate(
      225deg
    );
  transform: rotate(
      225deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-13 {
  -webkit-transform: rotate(
      247.5deg
    );
  -moz-transform: rotate(
      247.5deg
    );
  -ms-transform: rotate(
      247.5deg
    );
  transform: rotate(
      247.5deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-14 {
  -webkit-transform: rotate(
      270deg
    );
  -moz-transform: rotate(
      270deg
    );
  -ms-transform: rotate(
      270deg
    );
  transform: rotate(
      270deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-15 {
  -webkit-transform: rotate(
      292.5deg
    );
  -moz-transform: rotate(
      292.5deg
    );
  -ms-transform: rotate(
      292.5deg
    );
  transform: rotate(
      292.5deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-16 {
  -webkit-transform: rotate(
      315deg
    );
  -moz-transform: rotate(
      315deg
    );
  -ms-transform: rotate(
      315deg
    );
  transform: rotate(
      315deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

.emotion-17 {
  -webkit-transform: rotate(
      337.5deg
    );
  -moz-transform: rotate(
      337.5deg
    );
  -ms-transform: rotate(
      337.5deg
    );
  transform: rotate(
      337.5deg
    );
  display: inline-block;
  position: absolute;
  transform-origin: 0 100px;
  top: -100px;
}

<div>
  <div
    class="emotion-0"
  >
    <div
      class="emotion-1"
      style="transform: rotate(0deg);"
    >
      <span
        class="emotion-2"
      >
        A
      </span>
      <span
        class="emotion-3"
      >
        u
      </span>
      <span
        class="emotion-4"
      >
        r
      </span>
      <span
        class="emotion-5"
      >
        e
      </span>
      <span
        class="emotion-6"
      >
        l
      </span>
      <span
        class="emotion-7"
      >
        i
      </span>
      <span
        class="emotion-8"
      >
        a
      </span>
      <span
        class="emotion-9"
      >
        ✨
      </span>
      <span
        class="emotion-10"
      >
        K
      </span>
      <span
        class="emotion-11"
      >
        o
      </span>
      <span
        class="emotion-12"
      >
        r
      </span>
      <span
        class="emotion-13"
      >
        b
      </span>
      <span
        class="emotion-14"
      >
        e
      </span>
      <span
        class="emotion-15"
      >
        l
      </span>
      <span
        class="emotion-16"
      >
        a
      </span>
      <span
        class="emotion-17"
      >
        ✨
      </span>
    </div>
  </div>
</div>
`;
